h/migrations/versions/8f5b6f8bbba8_add_token_user_id_column.py

```diff
@@ -0,0 +1,20 @@
+"""Add token.user_id column."""
+
+from alembic import op
+from sqlalchemy import Column, ForeignKey, Integer
+
+revision = "8f5b6f8bbba8"
+down_revision = "aeda42d241b3"
+
+
+def upgrade():
+    op.add_column(
```

Comment:
I've made a comment on the model PR about an index for this column.

Reply:
Added the `index=True` here as well